.cirrus/tasks_gcp_marketplace.yml

```diff
@@ -62,6 +62,7 @@ verify_gcp_staging_app_task:
 
 release_gcp_prod_app_task:
   <<: *BUILD_GCP_APP_TEMPLATE
+  skip: $TRIGGER != "PUSH_PROD_DEPLOYER"
```

Comment:
WDYT about adding a comment here about you overriding the skip from `BUILD_GCP_APP_TEMPLATE`? It took me longer than it should've to understand why you had to add the `skip` here if the `only_if` already existed.

What about removing the `only_if` as well?

Reply:
fair points